main.go

```diff
@@ -91,13 +99,21 @@ func main() {
 	// See https://github.com/improbable-eng/etcd-cluster-operator/issues/171
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	validatedInterval, err := interval.ValidateForDefrag(defragWithoutThresholdInterval)
+	if err != nil {
+		setupLog.Error(err, "invalid defrag-without-threshold-interval configuration")
```

Comment:
It might be worth printing out the supplied value in this error message

Reply:
It is part of the error we are getting e.g.
```
ERROR   setup   invalid defrag-without-threshold-interval configuration {"error": "invalid interval: every 5s"}
```
Should we move it up to log it as key:value here as well?

Reply:
Ah yep, I misread. 

Ignore me \:)